Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/Contrast-Security-OSS/vulneruby
-  revision: b6d43bf42dfe09490c032600937eddf323362026
+  revision: e657b1d6a42a5eef291e8102c5c528bf577db1c0
```

Comment:
do we need to bind to a revision or should we always take the latest from the main branch?

Reply:
I think for now since vulneruby isn't deployed anywhere keeping it in sync with master commit is fine.